core/server/common/src/main/java/alluxio/master/StateLockManager.java

```diff
@@ -142,6 +142,10 @@ public LockResource lockShared() throws InterruptedException {
           exclusiveOnlyRemainingMs, PropertyKey.Name.MASTER_BACKUP_STATE_LOCK_EXCLUSIVE_DURATION);
       throw new IllegalStateException(safeModeMsg);
     }
+    // Check if the active interrupt cycle is on, we do not try to obtain the shared lock
```

Comment:
@bf8086 should we remove this part for now, because this has not been observed? and probably would not happen because of fair locking?

Reply:
Makes sense.